### ISSUE TITLE
How to create an external index with this extension

### ISSUE THREAD

**Issue by ToBoMi** [clangd/vscode-clangd#411]
> is there already a way to tell clangd where additional source files are.

You can create `compile_commands.json` for external libraries and (if necessary) [point clangd to them using config file](https://clangd.llvm.org/config#compilationdatabase). You can even [create an index](https://clangd.llvm.org/config#external) for these libs, especially if you don't intend to edit them.

_Originally posted by @i-ky in https://github.com/clangd/vscode-clangd/issues/38#issuecomment-1315010576_
      
How do I do this?
It seems one needs the `clangd-indexer`. It should be included in a complete LLVM install. I do not have a full LLVM/clangd installation, because I need to gather administrative rights for the installation.
Is there another way?

**Comment by i-ky** [comment]
> It seems one needs the `clangd-indexer`.

Yes, see [here](https://clangd.llvm.org/guides/remote-index#obtaining-a-suitable-version-of-clangd).

**Comment by ToBoMi** [comment]
Great. Thanks. The clangd releases don't need to be installed at all. I thought I need a LLVM installation.
Should we leave this issue open to suggest a feature request: Creating an external index?
